src/pkg/blob/dao/dao.go

```diff
@@ -167,12 +172,34 @@ func (d *dao) UpdateBlob(ctx context.Context, blob *models.Blob) error {
 	return err
 }
 
-func (d *dao) ListBlobs(ctx context.Context, query *q.Query) ([]*models.Blob, error) {
-	qs, err := orm.QuerySetter(ctx, &models.Blob{}, query)
+func (d *dao) ListBlobs(ctx context.Context, params models.ListParams) ([]*models.Blob, error) {
+	qs, err := orm.QuerySetter(ctx, &models.Blob{}, nil)
 	if err != nil {
 		return nil, err
 	}
 
+	if len(params.BlobDigests) > 0 {
+		qs = qs.Filter("digest__in", params.BlobDigests)
```

Comment:
Replace all `__in` and other Beego ORM operator if any with the new query model

Reply:
I make `QuerySetter` support `xxx__in` key in `q.KeyWords`.